har/har_handlers.go

```diff
@@ -67,9 +69,33 @@ func (h *resetHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		log.Errorf("har: method not allowed: %s", req.Method)
 		return
 	}
-	h.logger.Reset()
+
+
+	v, err := parseBoolQueryParam(req.URL.Query(), "return")
+	if err != nil {
+		log.Errorf("har: invalid value for return param: %s", err)
+		rw.WriteHeader(http.StatusBadRequest)
+	}
+
+	if v {
+		rw.Header().Set("Content-Type", "application/json; charset=utf-8")
+		hl := h.logger.ExportAndReset()
+		json.NewEncoder(rw).Encode(hl)
+	} else {
+		h.logger.Reset()
+		rw.WriteHeader(http.StatusNoContent)
+	}
 
 	log.Infof("resetHandler.ServeHTTP: HAR logs cleared")
+}
 
-	rw.WriteHeader(http.StatusNoContent)
+func parseBoolQueryParam(params url.Values, name string) (bool, error) {
```

Comment:
Add a godoc comment to this func to explain that in the case that the value is unparseable, it returns `true, error` instead of the more idiomatic `false, error` (because `false` is the zero value of `bool`)

Reply:
Sorry, bad refactor, was supposed to return `false, error`